app/models/vacols/case_docket.rb

```diff
@@ -315,23 +375,48 @@ def self.distribute_nonpriority_appeals(judge, genpop, range, limit, bust_backlo
   end
   # rubocop:enable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity, Metrics/ParameterLists
 
+  def self.any_or_not_genpop?(genpop)
+    genpop == "any" || genpop == "not_genpop"
+  end
+
+  def self.any_or_only_genpop?(genpop)
+    genpop == "any" || genpop == "only_genpop"
+  end
+
+  def self.exclude_remands_tied_to_other_judges(judge)
+    appeals_to_exclude = VACOLS::CaseDocket.remand_appeals_in_affinity_for_other_judges(judge: judge)
+    "AND BFKEY NOT IN (#{appeals_to_exclude.join(', ')})" if appeals_to_exclude.any?
```

Comment:
```suggestion
    appeals_to_exclude = VACOLS::CaseDocket.remand_appeals_in_affinity_for_other_judges(judge: judge)
    
    "AND BFKEY NOT IN (#{appeals_to_exclude.join(', ')})" if appeals_to_exclude.any?
```

I think this would read a lot better with a blank line in between. Otherwise starting the second line with a quoted string feels jarring to me.